mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -57,6 +58,36 @@
 warnings.simplefilter('always')
 
 
+def test_vhdr_codepage_ansi():
+    """Test BV reading with ANSI codepage."""
+    raw_init = read_raw_brainvision(vhdr_path, event_id=event_id)
+    tempdir = _TempDir()
+    ansi_vhdr_path = op.join(tempdir, op.split(vhdr_path)[-1])
+    ansi_vmrk_path = op.join(tempdir, op.split(vmrk_path)[-1])
+    ansi_eeg_path = op.join(tempdir, op.split(eeg_path)[-1])
+    # copy data file
+    shutil.copy(eeg_path, ansi_eeg_path)
+    # modify header file
+    with open(ansi_vhdr_path, 'wb') as fout:
+        with open(vhdr_path, 'rb') as fin:
+            for line in fin:
+                # Common Infos section
+                if line.startswith(b'Codepage'):
+                    line = b'Codepage=ANSI\n'
+                fout.write(line)
+    # modify marker file
+    with open(ansi_vmrk_path, 'wb') as fout:
+        with open(vmrk_path, 'rb') as fin:
+            for line in fin:
+                # Common Infos section
+                if line.startswith(b'Codepage'):
+                    line = b'Codepage=ANSI\n'
+                fout.write(line)
+    raw = read_raw_brainvision(ansi_vhdr_path, event_id=event_id)
+    data_new, times_new = raw[:]
```

Comment:
@agramfort I did use git blame. that's why I ping you. You add it. And I think that you only wanted to test `ch_names` in the next line.